README.md

```diff
@@ -1,233 +1,247 @@
-  "created_at": "2020-12-17T01:26:10Z",
-  "remaining": [
-    0,
-    1
+  "project_name": "Trifid SRO Filters",
+  "scheduled_with_events": [],  // Associated calendar reservations
+  "created_at": "2022-04-26T00:56:09Z",
+  "remaining": [  // Number of exposures left to take, for each set of exposures
+    "3",
+    "30"
   ],
-  "project_note": "for adding data",
-  "project_data": [
-    [
-      "file-blue-1",
-      "file-blue-1"
-    ],
-    [
-      "file-red-1",
-      "file-red-1",
-      "file-red-1"
-    ]
+  "project_note": "SRO filters",
+  "project_data": [  // Filenames of completed exposures
+    [],
+    []
   ],
-  "exposures": [
+  "project_sites": [  // Which site to observe or auto-schedule at (planned)
+    "sro"
+  ],
+  "exposures": [  // Details of each exposure request
     {
-      "filter": "Blue",
       "area": "FULL",
-      "exposure": 1,
+      "filter": "PL",
       "dither": "no",
-      "bin": "2, 2",
-      "count": "2",
+      "exposure": "15",
+      "bin": "1, 1",
+      "count": "3",
       "photometry": "-",
       "imtype": "light",
       "defocus": 0
     },
+
     {
-      "filter": "Red",
       "area": "FULL",
-      "exposure": 1,
+      "filter": "HA",
       "dither": "no",
-      "bin": "2, 2",
-      "count": "4",
+      "exposure": "120",
+      "bin": "1, 1",
+      "count": "30",
       "photometry": "-",
       "imtype": "light",
       "defocus": 0
-    }
+    },
   ],
-  "project_targets": [
+  "project_targets": [  // Target name, right ascension, and declinations
     {
-      "name": "M 1",
-      "ra": "5.5755",
-      "dec": "22.0145"
+      "name": "M 20",
+      "dec": "-22.9717",
+      "ra": "18.0450"
     }
   ]
 }
 ```
+
+### Request Parameters
+
+**project_name** and **created_at**: Projects are uniquely identified by the combination of their name and creation date.
+
+**exposure_index**: The exposure_index value specifies which of the multiple requested exposures was used for the completed file. 
+
+**base_filename**: This is the general filename that is used to get the images completed in a project. Base filenames currently follow a "sitecode-camera_name-date-exp_number" format, such as "saf-sq002me-20220615-00022012", with a reminder that this should not include the EX00 value or the .fits file extensions.
+
+**target_index (currently not supported)**: When projects used to support multiple targets, the target index behaved like the exposure index, and the project_data had one more level of nesting. Files would save at `project_data[target_index][exposure_index]`. We have opted to remove this feature; however, multi-target project support may be added again in the future.
+
+## API Endpoints
+Project requests are handled at the base URL `https://projects.photonranch.org/{stage}`, where `{stage}` is the deployment stage in ["test", "dev", "prod"]. Currently, both the production and development stages point to the dev URL, though this will change in the future.
+
```

Comment:
I don't think we have a full decision on this yet unfortunately...I personally think switching all of the production URLs to end in the service name (like "projects" instead of "prod") makes the most sense, but I think the biggest barrier to this is having to migrate any tables with the old name and making sure it's consistent across everything. I think that's why it still remains inconsistent across our repositories. Maybe @timbeccue has a better answer to this.

Reply:
I've been thinking about this too, and currently leaning towards just using 'prod' across the board for the production environments. That makes it a lot easier to pass the stage directly into the url instead of doing a lookup to translate 'prod' to 'calendar', or 'projects', or whatever service is actually being used. 

The two barriers to that are table migrations and changing Wayne's code, neither of which is too complicated. For the migrations you can use that script I wrote which I'll include somewhere, and for Wayne you'll just have to coordinate a time when he can update the urls on his end of things without disrupting ongoing activity.